Makefile

```diff
@@ -140,7 +141,7 @@ Test%:
 	kubectl -n numaflow-system delete po -lapp.kubernetes.io/component=controller-manager,app.kubernetes.io/part-of=numaflow
 	kubectl -n numaflow-system delete po e2e-api-pod  --ignore-not-found=true
 	cat test/manifests/e2e-api-pod.yaml |  sed 's@quay.io/numaproj/@$(IMAGE_NAMESPACE)/@' | sed 's/:latest/:$(VERSION)/' | kubectl -n numaflow-system apply -f -
-	-go test -v -timeout 15m -count 1 --tags test -p 1 ./test/$(shell grep $(*) -R ./test | head -1 | awk -F\/ '{print $$3}' ) -run='.*/$*'
+	-go test -v -timeout 15m -count 1 --tags test -p 1 ./test/api-e2e -run='.*/$*'
```

Comment:
@jy4096 - revert it.

Does it really take a long time to run the shell script to location the directory?

Reply:
> @jy4096 - revert it.
> 
> Does it really take a long time to run the shell script to location the directory?

no..

I think initially we did this because some tests have the same name?..